unittests/dataproviders/status.dataprovider.php

```diff
@@ -0,0 +1,74 @@
+<?php
+namespace Dataprovider;
+
+class Status {
+    public static function getStatusStringsProvider()
```

Comment:
In this special case *I* would probably have used a non-static method. Just because this method uses two other methods from the same class. But as there is no state within the class `static` is fine as well. And the 2  called methods are declared private so they aren't visible somewhere else and everything is fine.